src/applications/check-in/travel-claim/pages/validate/index.jsx

```diff
@@ -63,9 +63,7 @@ const Validate = ({ router }) => {
     [goToNextPage, lastName, dob, dobError, setSession, token, updateError],
   );
 
-  const validateErrorMessage = t(
-    'sorry-we-couldnt-find-an-account-that-matches-last-name-or-dob',
-  );
+  const validateErrorMessage = t('were-sorry-we-couldnt-find-an-account');
```

Comment:
why the new key?

Reply:
One of the `’` was a `'` so when I did a search I didn't find a match and assumed it was new. I reverted back to the old one and removed the new key and string.